package.json

```diff
@@ -7,7 +7,9 @@
     "show:report": "npx http-server ./playwright-report",
     "test": "CATEGORY=all npx playwright test",
     "test:codegen": "npx playwright codegen",
-    "test:chromium": "CATEGORY=allnpx playwright test --project=chromium",
+    "test:chrome": "CATEGORY=all-chrome npx playwright test --project='Desktop - Chromium'",
+    "test:firefox": "CATEGORY=all-firefox npx playwright test --project='Desktop - Firefox'",
+    "test:webkit": "CATEGORY=all-webkit npx playwright test --project='Desktop - Webkit'",
```

Comment:
q: do we need these? scripts block has a lot of options...i dont know what to choose if i am to come in here without context.. i challenge you to pare some of these down instead of adding.